src/components/BugBountyCards.tsx

```diff
@@ -1,154 +1,105 @@
+import { BaseHTMLAttributes } from "react"
 import { useTranslation } from "next-i18next"
-import {
-  Box,
-  type BoxProps,
-  Center,
-  type CenterProps,
-  Divider,
-  Flex,
-  type FlexProps,
-  Heading,
-  type HeadingProps,
-  type TextProps,
-  useToken,
-} from "@chakra-ui/react"
 
 import type { ChildOnlyProp, TranslationKey } from "@/lib/types"
 
-import { ButtonLink, type ButtonLinkProps } from "@/components/Buttons"
-import Text from "@/components/OldText"
+import { cn } from "@/lib/utils/cn"
+
+import { ButtonLink, ButtonLinkProps } from "./ui/buttons/Button"
+import { Center, Flex, Stack } from "./ui/flex"
+
+type FlexProps = BaseHTMLAttributes<HTMLDivElement>
 
 const CardRow = ({ children }: ChildOnlyProp) => (
-  <Flex justifyContent="space-between" my="16" mx="4" flexWrap="wrap">
-    {children}
-  </Flex>
+  <Flex className="mx-4 my-16 flex-wrap justify-between">{children}</Flex>
 )
 
 const SubmitBugBountyButton = ({ children, ...props }: ButtonLinkProps) => (
-  <ButtonLink m="4" href="https://forms.gle/Gnh4gzGh66Yc3V7G8" {...props}>
+  <ButtonLink
+    className="m-4"
+    href="https://forms.gle/Gnh4gzGh66Yc3V7G8"
+    {...props}
+  >
     {children}
   </ButtonLink>
 )
 
 const Card = ({ children, ...props }: FlexProps) => {
-  const tableBoxShadow = useToken("colors", "tableBoxShadow")
-
   return (
-    <Flex
-      flexDir="column"
-      flex={{ base: "1 1 412px", xl: "1 1 260px" }}
-      justifyContent="space-between"
-      bg="background.base"
-      borderRadius="2px"
-      borderRad
-      boxShadow={tableBoxShadow}
-      border="1px solid"
-      borderColor="border"
-      m="4"
-      _hover={{
-        borderRadius: "base",
-        boxShadow: "tableBoxHover",
-        background: "tableBackgroundHover",
-        transition: "transform 0.1s",
-        transform: "scale(1.02)",
-      }}
+    <Stack
+      className={cn(
+        "flex-[1_1_412px] gap-0 xl:flex-[1_1_216px]",
+        "m-4 justify-between bg-background",
+        "rounded-sm border border-[#e5e5e5] shadow-table-box dark:border-[#333]",
```

Comment:
Lets replace this with the default border color.
```suggestion
        "rounded-sm border shadow-table-box",
```